### PR TITLE
Update metadata, font

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
Update metadata and font for Ukrainian and Russian localization.

### CODE REVIEW

File: Langs/0419.lang [hunk 1]
```diff
@@ -1,7 +1,12 @@
 [MetaData]
 Language=Russian - Russia
 Translator=c0de_6 (@Agc0de6)
-Version=2.2.5.0
+Version=2.3.0.3
```
Comment (micwoj92) [inline Langs/0419.lang:4]: needs to be 2.3.0.4 in both now

File: Langs/0419.lang [hunk 2]
```diff
@@ -1,7 +1,12 @@
 [MetaData]
 Language=Russian - Russia
 Translator=c0de_6 (@Agc0de6)
```
Comment (micwoj92) [inline Langs/0419.lang:3]: could you update this so that only github username stays?

File: Langs/0422.lang [hunk 3]
```diff
@@ -1,7 +1,12 @@
 [MetaData]
 Language=Ukrainian - Ukraine
 Translator= Oleg Chashko
```
Comment (micwoj92) [inline Langs/0422.lang:3]: Here too, please update to only GH username

**Review by micwoj92 — CHANGES_REQUESTED** [review]

**Comment by micwoj92** [comment]
Also for the future, could you use branches or delete and then fork again? Otherwise this looks messy and when commiting changes it includes then by default :P 
image

**Comment by Oleg-Chashko** [comment]
@micwoj92 
_Also for the future, could you use branches or delete and then fork again? Otherwise this looks messy and when commiting changes it includes then by default :P_

Sorry bro. I have no idea how to do this. :(

**Comment by micwoj92** [comment]
Go to your repo, click on settings, scroll down and then click on delete and confirm.

When you have another fix you can clone this repo again and then you will have fresh copy.

**Comment by Oleg-Chashko** [comment]
> Go to your repo, click on settings, scroll down and then click on delete and confirm.
> 
> When you have another fix you can clone this repo again and then you will have fresh copy.

Thank you. Did as you described. But it seems to me that this is not the best way?
Is there a better way to create branches?

**Comment by micwoj92** [comment]
Oh that was how to delete repo. If you want to create new branch for pull request it's easiest to just go to one file on this repo, for example https://github.com/rcmaehl/WhyNotWin11/blob/main/Langs/0419.lang

Then you click on edit (pencil emote)
and after you are done it will do following:
create new branch (patch-1 name by default)
automatically forward you to pull request

**Comment by Oleg-Chashko** [comment]
@micwoj92 
Thank you! :)

**Pull request merged**